packages/core/src/createProps.ts

```diff
@@ -107,7 +107,7 @@ export const getPropTable = (): [string, string, string, CCSSParser?, CCSSParser
         ['maxH', 'max-hei', 'max-height', mapValue, parseSingle],
 
         // Grid
-        ['g', 'gr', 'grid'],
+        ['g', 'grd', 'grid'],
```

Comment:
I renamed it because we had two entries with the same `gr` name. This was causing issues after creating the interface.

Reply:
Good, I was afraid we will have such :)